conda/recipes/librmm/meta.yaml

```diff
@@ -51,8 +51,16 @@ outputs:
     requirements:
       build:
         - cmake {{ cmake_version }}
+        # We require spdlog and fmt (which was devendored from spdlog
+        # conda-forge packages in 1.11.0) so that the spdlog headers are not
+        # pulled by CPM and installed as a part of the rmm packages. However,
+        # building against librmm still requires these headers so they are also
+        # exported as a run requirement.
+        - fmt {{ fmt_version }}
+        - spdlog {{ spdlog_version }}
       run:
         - cudatoolkit {{ cuda_spec }}
+        - fmt {{ fmt_version }}
         - spdlog {{ spdlog_version }}
```

Comment:
I think we can rely on the `run_exports` of these packages instead of explicitly adding them to the `run` section here.
- https://github.com/conda-forge/fmt-feedstock/blob/main/recipe/meta.yaml#L16-L17
- https://github.com/conda-forge/spdlog-feedstock/blob/main/recipe/meta.yaml#L15-L16

Reply:
Good call! Removed from the run section in 8d4d42a055159bf0889431584b20b6b4342d3f8f.